assets/app/templates/nav.html

```diff
@@ -1,7 +1,7 @@
 <ul id="nav-mobile">
 <% if(username) {%>
   <li>hey <%- username %>!</li>
-  <li><a href="/logout">logout</a></li>
+  <li><a id="logout">logout</a></li>
```

Comment:
How are you using ids? I hope not for styling. I'd suggest `js` prefixing selectors for use in js.